### PR TITLE
feat: add governance to spiritnet runtime

### PR DESCRIPTION
**Pull request by wischli**
## fixes KILTProtocol/ticket#1579 + KILTProtocol/ticket#1593
* Adds governance to KILT Spiritnet
* Moves `fast-gov` consts to corresponding modules in `kilt-primitives::constants`
* Removes Spiritnet filter for `session.set_keys`

## Checklist:

- [x] I have verified that the code works
  - [x] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [x] I have verified that the code is easy to understand
- [x] This PR does not introduce new custom types
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: runtimes/spiritnet/Cargo.toml [hunk 1]
```diff
@@ -42,12 +42,18 @@ frame-system = {git = "https://github.com/paritytech/substrate", default-feature
...
 pallet-balances = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
+pallet-collective = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
+pallet-democracy = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
 pallet-indices = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
+pallet-membership = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
 pallet-randomness-collective-flip = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
+pallet-scheduler = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
 pallet-session = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
+# pallet-society = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.9"}
```
Comment (weichweich) [inline runtimes/spiritnet/Cargo.toml:52]: W: I don’t think That we will use the society pallet in the future. So maybe we just remove this line?

File: runtimes/spiritnet/Cargo.toml [hunk 2]
```diff
@@ -88,13 +95,18 @@ runtime-benchmarks = [
...
   "kilt-launch/runtime-benchmarks",
   "pallet-balances/runtime-benchmarks",
+  "pallet-collective/runtime-benchmarks",
+  "pallet-democracy/runtime-benchmarks",
   "pallet-indices/runtime-benchmarks",
-  "parachain-staking/runtime-benchmarks",
+  "pallet-membership/runtime-benchmarks",
+  "pallet-scheduler/runtime-benchmarks",
+  # "pallet-session/runtime-benchmarks",
+  # "pallet-society/runtime-benchmarks",
```
Comment (weichweich) [inline runtimes/spiritnet/Cargo.toml:104]: W: Same here

File: runtimes/spiritnet/Cargo.toml [hunk 3]
```diff
@@ -113,18 +126,25 @@ std = [
...
   "pallet-balances/std",
+  "pallet-collective/std",
+  "pallet-democracy/std",
   "pallet-indices/std",
+  "parachain-info/std",
+  "pallet-membership/std",
   "pallet-randomness-collective-flip/std",
+  "pallet-scheduler/std",
   "pallet-session/std",
+#  "pallet-society/std",
```
Comment (weichweich) [inline runtimes/spiritnet/Cargo.toml:139]: W: and here

File: runtimes/spiritnet/Cargo.toml [hunk 4]
```diff
@@ -142,24 +162,29 @@ std = [
...
   "pallet-authorship/try-runtime",
+  "pallet-balances/try-runtime",
+  "pallet-collective/try-runtime",
+  "pallet-democracy/try-runtime",
   "pallet-indices/try-runtime",
+  "pallet-membership/try-runtime",
   "pallet-randomness-collective-flip/try-runtime",
+  "pallet-scheduler/try-runtime",
   "pallet-session/try-runtime",
+#  "pallet-society/try-runtime",
```
Comment (weichweich) [inline runtimes/spiritnet/Cargo.toml:182]: W: here

**Review by weichweich — CHANGES_REQUESTED** [review]
We need to increase the runtime version before we merge.

File: runtimes/spiritnet/Cargo.toml (hunk 1 above)
Comment (wischli) [reply runtimes/spiritnet/Cargo.toml:52]: Yes I agree. I actually intended to remove it everywhere but forgot about it midway.

File: runtimes/spiritnet/Cargo.toml (hunk 4 above)
Comment (wischli) [reply runtimes/spiritnet/Cargo.toml]: e08ad89

**Review by weichweich — APPROVED** [review]
LGTM

**Pull request merged**